common/com_model.h

```diff
@@ -109,7 +109,7 @@ typedef struct glpoly_s
 	struct glpoly_s	*chain;
 	int		numverts;
 	int		flags;          		// for SURF_UNDERWATER
-	float		verts[4][VERTEXSIZE];	// variable sized (xyz s1t1 s2t2)
+	float		(*verts)[VERTEXSIZE];	// variable sized (xyz s1t1 s2t2)
```

Comment:
I see. Sorry about that. I found simple solution without macros and modify structs.